Update scRNA-seq chapter and add a paragraph on quantification of gene expression (#371)
theislab/single-cell-best-practices: jupyter-book/introduction/scrna_seq.md

Two general types of cells exist: eukaryotes and prokaryotes.
Eukaryotic cells contain a nucleus, where the nuclear membrane encapsulates the chromosomes; while prokaryotic cells only have a nucleoid region but no nucleus.
The nucleus hosts the cells' genomic DNA, which is why they are called eukaryotes: _Nucleus_ is Latin for kernel or seed.
The DNA replication machinery reads the hereditary information that is stored in the DNA in the nucleus to replicate themselves and keep the life cycle going.
The eukaryotic DNA is divided into several linear bundles called chromosomes, which are separated by the microtubular spindle during nuclear division.
Understanding the hereditary information hidden in DNA is key to understanding many evolutionary and disease-related processes.

_Sequencing_ is the process of deciphering the order of DNA nucleotides.
It is primarily used to unveil the genetic information that is carried by a specific DNA segment, a complete genome, or even a complex microbiome.

> DNA _sequencing_ allows researchers to identify a gene's location, function, and regulation. For example, it uncovers genetic features such as open reading frames (ORFs) or {term}`CpG` islands, which indicate promotor regions.

Reviewer: Not sure whether we need DNA sequencing and above sequencing to be italic. We may as well just have a glossary item.

Reviewer: ORFs can be a glossary item.

Author: I removed the italics and added sequencing to the glossary.

However, technical limitations in current scRNA-seq technologies further complicate measurement. For example, popular platforms like 10X Genomics capture only up to 65% of cells per run and recover just ~14% of each cell’s mRNA {cite}`aljanahi2018introduction`. These constraints make it especially challenging to detect weakly expressed genes.

Understanding the central dogma through these numerical lenses reveals not only the biological complexity but also the limitations of our tools. To appreciate this more deeply, let’s walk step by step from gene to protein.

#### From Gene to Protein

Our journey begins with a gene, a defined region in the DNA that acts as a template for mRNA synthesis.
While the number of genes can vary slightly between individuals (~70 genes), the average human genome contains roughly 22,000 genes {cite}`pertea2010between`.
Gene transcription is far from continuous.
Instead, it occurs in stochastic bursts — short, irregular periods of activity during which a gene might suddenly produce multiple mRNA transcripts before returning to silence {cite}`suter2011mammalian`.

> This is also the reason why we model mRNA transcription with a negative binomial distribution.

Reviewer: The NBB might need 0.5 sentences of explanation here. Not where it's coming from, because you are nicely doing that, but its properties

Author: Added two useful properties from negative binomial distribution.

- **Separation in droplets**: These methods encapsulate individual cells into tiny droplets within an emulsion, enabling high-throughput processing.
- **Separation in physical compartments**: These techniques isolate cells into distinct physical compartments, often referred to as wells.

Each approach differs in the ability to recover transcripts, the number of sequenced cells, and many other aspects.
In the following subsections, we will briefly discuss how they work, their strengths and weaknesses, and possible biases that data analysts should be aware of regarding the respective protocols.

#### Separation in Droplets

##### Most common protocols

The most widely used protocols are **inDrop** {cite}`Klein2015`, **Drop-seq** {cite}`exp:Macosko2015` and the commercially available **10x Genomics Chromium** {cite}`exp:Zheng2017`.
These protocols use microfluids to trap cells in tiny, watery bubbles known as droplets.
Each droplet forms a separate spaces containing only one cell and the required chemicals (beads).

> The mentioned protocols are able to generate droplets several thousand times per second.

Reviewer: ```suggestion
The mentioned protocols can to generate droplets thousands of times per second.
```

Nit: More idiomatic

Author: Adapted the sentence!
